docs/local.rst

```diff
@@ -141,6 +141,24 @@ The default Open edX theme is rather bland, so Tutor makes it easy to switch to
 
 Out of the box, only the default "open-edx" theme is available. We also developed `Indigo, a beautiful, customizable theme <https://github.com/overhangio/indigo>`__ which is easy to install with Tutor.
 
+Changing the mysql charset and collation
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. warning:: This change is potentially irreversible. It is recommended to make a backup of the MySQL database. See the :ref:`database dump instructions <database_dumps>` to create a DB dump.
+
+Your database's charset and collation might not support specific characters or emojis. To upgrade your charset and collation of all the tables in your database to utf8mb4 and utf8mb4_unicode_ci respectively, run::
+
+    tutor local do change-charset-collation --all-tables --charset=utf8mb4 --collation=utf8mb4_unicode_ci
```

Comment:
In which situation(s) will users have to run this command? Is it for users upgrading from Quince? Is it optional?

Reply:
Yes, they can run it after upgrading from Quince. It is optional as the LMS runs fine without it unless you add specific characters such as emojis that will throw DB errors.

Reply:
That explanation should be present in the docs :)